Updated guidance as more TIC3 work emerges

pburkholder opened the pull request:
## Changes proposed in this pull request:

- Note that things are changing, and CISA says folks should go forward with PaaS adoption.

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/peterb/tic3-initial-guidance)


## Security Considerations

Better guidance for navigating the gap between 2.0 and 3.0.

pburkholder committed:
Updated guidance as more TIC3 work emerges

pburkholder committed:
No quoting of CISA, and fmt fixes

mheadd commented on _docs/compliance/meeting-tic-requirements.md:
```diff
@@ -49,7 +64,7 @@ APIRouter -->|Change app| App
 Any traffic from an agency-authorized boundary (eg physical network of an agency building, or collective virtual boundary for all networked agency buildings) to one that it is not under your agency’s control (e.g. the open internet, or cloud.gov) is likely already routed over a TIC connection.
 
-For requests originating from your agency’s TIC egress bound for cloud.gov, our TLS implementation plays the role of the orange "encrypted tunnel" in Figure 14 of the [TIC reference architecture](https://s3.amazonaws.com/sitesusa/wp-content/uploads/sites/482/2015/04/TIC_Ref_Arch_v2-0_2013.pdf). You will access both api.fr.cloud.gov and yourapp.app.cloud.gov exclusively over TLS because the cloud. ...
+For requests originating from your agency’s TIC egress bound for cloud.gov, our TLS implementation plays the role of the orange "encrypted tunnel" in Figure 14 of the [TIC 2.0 reference architecture](https://s3.amazonaws.com/sitesusa/wp-content/uploads/sites/482/2015/04/TIC_Ref_Arch_v2-0_2013.pdf). You will access both api.fr.cloud.gov and yourapp.app.cloud.gov exclusively over TLS because the cl ...
```
When I try and access the PDF document linked in this section I get an error. Downloading by other means also results in an error. Not sure if it's me or the link (or doc) is broken.

mheadd commented on _docs/compliance/meeting-tic-requirements.md:
```diff
@@ -91,6 +106,6 @@ RouteService[App logic or route service] --> App
 {% endmermaid_diagram %}
 
-For remote workers or partners outside the normal agency network boundary, you can require use of a VPN to ensure that cloud.gov-bound traffic is routed over the agency network and TIC (shown in Figure 15 of the [TIC Reference architecture](https://s3.amazonaws.com/sitesusa/wp-content/uploads/sites/482/2015/04/TIC_Ref_Arch_v2-0_2013.pdf)).
+For remote workers or partners outside the normal agency network boundary, you can require use of a VPN to ensure that cloud.gov-bound traffic is routed over the agency network and TIC (shown in Figure 15 of the [TIC 2.0 Reference architecture](https://s3.amazonaws.com/sitesusa/wp-content/uploads/sites/482/2015/04/TIC_Ref_Arch_v2-0_2013.pdf)).
```
Same issue here. Can't seem to access the PDF document.

Commit:
Update meeting-tic-requirements.md

Modify link to reference architecture, denote that it's an external link in case of future breakage.
